examples/docs/en-US/table.md

```diff
@@ -1500,6 +1501,79 @@ Customize table column so it can be integrated with other components.
 ```
 :::
 
+### Table with custom header
```

Comment:
Update the missing Spanish and Chinese docs (You don't need to translate).

Reply:
I did manage to add Spanish, but couldn't find the right spot on Chinese, can you give me a hand on that?

Reply:
https://github.com/ElemeFE/element/blame/dev/examples/docs/zh-CN/table.md#L1540